src/test/java/sg/ncl/MainControllerTest.java

```diff
@@ -143,7 +141,8 @@ public void testPlanPage() throws Exception {
                 .andExpect(content().string(containsString("footer id=\"footer\"")));
     }
 
-    @Test
+    // page is taken out as of 22/9/2016
+    @Ignore
     public void testFuturePlanPage() throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Fix or remove this skipped unit test [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1607)